class.jetpack-connection-banner.php

```diff
@@ -182,6 +185,7 @@ public static function enqueue_connect_button_scripts() {
 				'connectInPlaceUrl'     => Jetpack::admin_url( 'page=jetpack#/setup' ),
 				'dashboardUrl'          => Jetpack::admin_url( 'page=jetpack#/dashboard' ),
 				'plansPromptUrl'        => Jetpack::admin_url( 'page=jetpack#/plans-prompt' ),
+				'identity'             => $identity,
```

Comment:
Nit: this seems to need an extra space.

```suggestion
				'identity'              => $identity,
```